### PR TITLE
Edit USERS-GUIDE.md

### PR DESCRIPTION
**Pull request by AnotherButler**
Make initial copy edits to file.

### CODE REVIEW

File: docs/USERS-GUIDE.md [hunk 1]
```diff
@@ -1,40 +1,41 @@
-# DAPLink Users Guide
-There are three interfaces that DAPLink provides. These are drag-n-drop programming, a serial port and debugging support. In addition DAPLink firmware can be updated using the drag-n-drop programming interface of the bootloader.
+# DAPLink users guide
 
+DAPLink provides three interface. These are drag-and-drop programming, a serial port and debugging support. In addition, you can update DAPLink firmware using the drag-and-drop programming interface of the bootloader.
```
Comment (loverdeg-ep) [inline docs/USERS-GUIDE.md]: interfaces?

File: docs/USERS-GUIDE.md [hunk 2]
```diff
@@ -1,40 +1,41 @@
-# DAPLink Users Guide
-There are three interfaces that DAPLink provides. These are drag-n-drop programming, a serial port and debugging support. In addition DAPLink firmware can be updated using the drag-n-drop programming interface of the bootloader.
+# DAPLink users guide
```
Comment (AnotherButler) [inline docs/USERS-GUIDE.md]: @brianesquilona Could you please add an introduction about what DAPLink is and why you'd use it?

Comment (brianesquilona) [reply docs/USERS-GUIDE.md]: @AnotherButler , can I redirect you to the readme file :) [The very first paragraph] (https://github.com/ARMmbed/DAPLink/blob/master/README.md)

Comment (AnotherButler) [reply docs/USERS-GUIDE.md]: Aha. I was looking at the README in docs, not in the overall repo.

File: docs/USERS-GUIDE.md [hunk 3]
```diff
@@ -1,40 +1,41 @@
...
 
-Note - Most DAPLink implementations support other baud rates in addition to the ones listed here.
+- 9600.
+- 14400.
+- 19200.
+- 28800.
+- 38400.
+- 56000.
+- 57600.
+- 115200.
```
Comment (flit) [inline docs/USERS-GUIDE.md]: Can you please remove the periods after the baud rates. That is a list of items, not sentences. It looks really weird with the period. Same for supported file formats above.

Comment (flit) [reply docs/USERS-GUIDE.md]: That's fine, and makes sense, for sentences. But these are not sentences here. I've *never* seen technical documentation that uses periods in cases like this.

**Pull request merged**